### PR TITLE
DDF-2553 updated documentation for 2.9.3 release

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds documentation for version 2.9.3 to `gh-pages` branch / codice.org/ddf

#### Who is reviewing it?

@codice/docs 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- confirm links, updated version on the documentation

#### What are the relevant tickets?

[DDF-2553](https://codice.atlassian.net/browse/DDF-2553)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -31940,15 +31961,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
...
-                    
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
+
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.3, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>{directory}/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
```
Comment (coyotesqrl) [inline documentation.html]: Nope. Definitely not my home directory. :)

File: extending.html [hunk 2]
```diff
@@ -2224,15 +2224,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
...
                     --&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
 
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.3, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>{directory}/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
```
Comment (coyotesqrl) [inline extending.html]: Again, nope.

File: documentation.html [hunk 3]
```diff
@@ -31940,15 +31961,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
-                    
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
```
Comment (coyotesqrl) [inline documentation.html]: Nope.

File: extending.html [hunk 4]
```diff
@@ -2224,15 +2224,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/rporter/Temp/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
```
Comment (coyotesqrl) [inline extending.html]: Nope.

File: documentation.html (hunk 1 above)
Comment (ricklarsen) [inline documentation.html]: I know this was fixed.

**Pull request closed without merging**